dashboard/app/jobs/evaluate_rubric_job.rb

```diff
@@ -0,0 +1,47 @@
+class EvaluateRubricJob < ApplicationJob
+  queue_as :default
+
+  rescue_from(StandardError) do |exception|
+    if rack_env?(:development)
+      puts "EvaluateRubricJob Error: #{exception.full_message}"
+    end
+    raise
+  end
+
+  # 2D Map from unit name and level name, to the name of the lesson files in S3
+  # which will be used for AI evaluation.
+  # TODO: This is a temporary solution. After the pilot, we should at least make
+  # the S3 pointer editable on levelbuilder, and eventually make all of the data
+  # it points to editable there too.
+  UNIT_AND_LEVEL_TO_LESSON_S3_NAME = {
+    'csd3-2023' => {
+      'CSD U3 Interactive Card Final_2023' => 'CSD-2022-U3-L17',
+      'CSD U3 Sprites scene challenge_2023' => 'New-U3-2022-L10',
+      'CSD web project animated review_2023' => 'New-U3-2022-L13',
+      'CSD games sidescroll review_2023' => 'New-U3-2022-L20'
+    }
+  }
+
+  def perform(user_id:, script_level_id:)
+    user = User.find(user_id)
+    script_level = ScriptLevel.find(script_level_id)
+    lesson_s3_name = EvaluateRubricJob.get_lesson_s3_name(script_level)
+    if rack_env?(:development)
+      puts "Evaluating rubric for user #{user.id} on script level #{script_level.id} with lesson_s3_name: #{lesson_s3_name.inspect}"
+    end
+
+    raise "lesson_s3_name not found for script_level_id: #{script_level.id}" if lesson_s3_name.blank?
+
+    puts "TODO: Implement the rest of this job"
```

Comment:
Do we repeat any check for AI disabling, here? It might suffice to just check before creating the job that the particular teacher has not disabled things in the meantime.

Reply:
added a note to https://codedotorg.atlassian.net/browse/AITT-246 . I think we can probably get away without this, since it seems reasonable for it to take a minute from the time they opt out until the time we stop sending their data to openai (though they would immediately stop seeing any ai data in the UI)

Reply:
I meant to say-- but on the other hand, this check is easy to implement so I'm also fine with checking a second time